cmd/ebrelayer/main.go

```diff
@@ -401,6 +415,16 @@ func RunInitWitnessCmd(cmd *cobra.Command, args []string) error {
 		return errors.New("validator moniker is empty")
 	}
 
+	sifnodeGrpc, err := cmd.Flags().GetString(sifnodeGrpcEntryPointFlag)
+	if err != nil {
+		// set default value
+		sifnodeGrpc = defaultSifnodeGrpc
+	}
+
+	if len(sifnodeGrpc) == 0 {
```

Comment:
When will sifnodeGrpc be 0 length?

Reply:
input is a empty string like --sifnode-grpc ""